recipes/cyclus/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "1.4.0rc1" %}
```

Comment:
Normally we don't start with an rc, but I'm not sure the earlier versions will build for us. So I'm fine with this.